content/en/docs/Writing policies/preconditions.md

```diff
@@ -146,9 +146,15 @@ The following operators are currently supported for precondition evaluation:
 - GreaterThanOrEquals
 - LessThan
 - LessThanOrEquals
+- DurationGreaterThan
+- DurationGreaterThanOrEquals
+- DurationLessThan
+- DurationLessThanOrEquals
 
 The set operators, `In` and `NotIn` support a set of strings as the value (e.g. In ["str1", "str2"]). They also allow you to specify a set of strings as the key (e.g. ["str1", "str2"] In ["str1", "str2", "str3"]). In this case `In` checks if **all** the strings part of the key are in the value set (i.e. key is a subset of value) and `NotIn` checks if **any** of the strings part of the key is **not** in the value set (i.e. key is not a subset of value). Sets of other types are currently not supported.
 
+Duration operators expect numeric key or value as seconds or as a string that is a valid Go time duration, eg: "1h".
```

Comment:
I feel this section could use another sentence or two. For example, what's the purpose of these? What's a use case? You show an example where the hours unit is used (i.e., "1h") but not seconds, minutes, days, months (?), or longer? Need to explicitly state the units supported by this operator.

Reply:
Added a few more sentences. I purposely didn't mention the units less than 1 second because didn't think it made sense given the integer value of `1` is converted to seconds. I did link to the Godoc for where `time.ParseDuration` documents the full list of supported units.